geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/DataSerializerPropagationDUnitTest.java

```diff
@@ -220,111 +192,111 @@ public boolean stopWaiting() {
     Wait.waitForCriterion(wc, 60 * 1000, 1000, true);
   }
 
-  public static void registerDSObject1() throws Exception {
+  private static void registerDSObject1() throws Exception {
     try {
       InternalDataSerializer.register(DSObject1.class, true);
     } catch (Exception e) {
       Assert.fail("Test failed due to exception in DSObject1", e);
     }
   }
 
-  public static void registerDSObject2() throws Exception {
+  private static void registerDSObject2() throws Exception {
     try {
       InternalDataSerializer.register(DSObject2.class, true);
     } catch (Exception e) {
       Assert.fail("Test failed due to exception in DSObject2", e);
     }
   }
 
-  public static void registerDSObject3() throws Exception {
+  private static void registerDSObject3() throws Exception {
     try {
       InternalDataSerializer.register(DSObject3.class, true);
     } catch (Exception e) {
       Assert.fail("Test failed due to exception in DSObject2", e);
     }
   }
 
-  public static void registerDSObject4() throws Exception {
+  private static void registerDSObject4() throws Exception {
     try {
```

Comment:
It would be nice to get rid of all of these try/catch statements.

Reply:
Agreed. And while I'm at it, I think I might just inline all these calls.